core/test/financial-templates/ExpiringMultiPartyCreator.js

```diff
@@ -52,6 +49,11 @@ contract("ExpiringMultiParty", function(accounts) {
       sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
       disputerDisputeRewardPct: { rawValue: toWei("0.1") }
     };
+
```

Comment:
Thank you for adding this. Like @mrice32 said yesterday master's code may have changed due to multiple PR's at the same time; I definitely had this in my version initially but all good that it's here now.